src/Stratis.Bitcoin.Features.Wallet.Tests/WalletManagerTest.cs

```diff
@@ -377,7 +377,7 @@ public void RecoverWalletWithEqualInputAsExistingWalletRecoversWallet()
                 dataFolder, new Mock<IWalletFeePolicy>().Object, new Mock<IAsyncLoopFactory>().Object, new NodeLifetime(), DateTimeProvider.Default, new ScriptAddressReader());
 
             // Try to recover it.
-            Wallet recoveredWallet = walletManager.RecoverWallet(password, walletName, deletedWallet.mnemonic.ToString(), DateTime.Now.AddDays(1), passphrase);
+            Wallet recoveredWallet = walletManager.RecoverWallet(password, walletName, deletedWallet.mnemonic.ToString(), DateTimeProvider.Default.GetUtcNow().AddDays(1), passphrase);
```

Comment:
Ideally you should use the same IDateTimeProvider:
```
IDateTimeProvider datetimeProvider = DateTimeProvider.Default;
var walletManager = new WalletManager(this.LoggerFactory.Object, KnownNetworks.StratisMain, chain, NodeSettings.Default(), new Mock<WalletSettings>().Object,
                dataFolder, new Mock<IWalletFeePolicy>().Object, new Mock<IAsyncLoopFactory>().Object, new NodeLifetime(), datetimeProvider, new ScriptAddressReader());

// Try to recover it.
Wallet recoveredWallet = walletManager.RecoverWallet(password, walletName, deletedWallet.mnemonic.ToString(), datetimeProvider.GetUtcNow().AddDays(1), passphrase);
```

same in the other places.

Reply:
I feel that this is a lot of changes for no benefit.